lib/loadEnv.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const dotenv = require('dotenv');
+const path = require('path');
+const ServerlessError = require('./classes/Error').ServerlessError;
+
+const isMissingFileError = error => error.code === 'ENOENT';
+
+const throwDotenvError = (error, filePath) => {
+  const errorMessage = [
+    `Encountered: "${error}" while trying to load environment variables`,
+    ` from filepath: ${filePath}.`,
+  ].join('');
+  throw new ServerlessError(errorMessage);
+};
+
+module.exports = async stage => {
+  const defaultEnvFilePath = path.join(process.cwd(), '.env');
+  const stageEnvFilePath = path.join(process.cwd(), `.env.${stage}`);
+
+  const { error: stageEnvResultError } = dotenv.config({ path: stageEnvFilePath });
+
+  if (!stageEnvResultError) return;
```

Comment:
Here, as suggested, I changed the approach to try to load files and based on `error` returned by `dotenv.config` decide how to proceed further. I'd love to hear your opinion on the chosen approach and I'm open to suggestions on how to improve it further

Reply:
It looks great!